com.adamcake.Bolt.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.adamcake.Bolt
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: bolt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --env=PATH=/app/bin:/usr/bin:/app/jre/bin
+modules:
+  - name: python-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install . --prefix=${FLATPAK_DEST}
+    build-options:
+      build-args:
+        - --share=network
+    sources:
+      - type: archive
+        url: https://github.com/python-pillow/Pillow/archive/refs/tags/10.0.0.tar.gz
+        sha256: 535d17e830427bec163027114ded1def9ab0350c99bf1d8cb10535032967f3a5
+  - name: bolt
```

Comment:
usually the module representing the main app is last in the list.

Reply:
Done: 2dc2525